.github/workflows/PR_validation.yml

```diff
@@ -555,3 +555,22 @@ jobs:
     - name: Remove Env
       if: ${{ always() }}
       uses: ./.github/actions/remove-env
+
+  Orca-ExampleTest-Ray-Py37-Spark3:
```

Comment:
put them in Line 425? UT order: basic -> pytest -> tutorial -> example

Reply:
it is fine i think. I will revert this back after actions pass.